docs/_posts/2024-05-18-leaflet-2.0.0-alpha.md

```diff
@@ -0,0 +1,199 @@
+4. To have global access to variables of a `module-script`, assign them to the `window` object (Not recommended): `window.map = map` 
+5. Consider [Leaflet V1 Polyfill](https://github.com/Falke-Design/Leaflet-V1-polyfill) if you are using outdated plugins
+
+
+### Old implementation
+
+```
+<script src="https://unpkg.com/leaflet@1.9.4/dist/leaflet.js"></script>
+<script>
+	const map = L.map('map').setView([51.505, -0.09], 13);
+
+	const tiles = L.tileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png', {
+		maxZoom: 19,
+		attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
+	}).addTo(map);
+</script>
+```
+
+### New implementation
+
+#### Module
+
+```
+<script type="importmap">
+	{
+		"imports": {
+			"leaflet": "https://unpkg.com/leaflet@2.0.0-alpha1/dist/leaflet.js"
+		}
+	}
+</script>
+<script type="module">
+	import L, {Map, TileLayer, Marker, Circle, Polygon, Popup} from 'leaflet';
+
+	const map = new Map('map').setView([51.505, -0.09], 13);
+
+	const tiles = new TileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png', {
+		maxZoom: 19,
+		attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
+	}).addTo(map);
+</script>
+```
+
+#### Global Script
+```
+<script src="https://unpkg.com/leaflet@2.0.0-alpha1/dist/leaflet-global.js"></script>
+<script>
+	const map = new L.Map('map').setView([51.505, -0.09], 13);
+
+	const tiles = new L.TileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png', {
+		maxZoom: 19,
+		attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
+	}).addTo(map);
+</script>
+```
+
+
+---
+
+## Major Changes & Cleanups
+
+### Modules & Browser Support
+- Dropped UMD and global `L` in favor of modern ESM
+- Targeting only **evergreen browsers**
+- Rewritten using [standardized ES6 classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes)
+- Removed legacy `SVG.VML` renderer
+- ... as well as various other ES6 features
+
+### Browser
+- Removed detection flags: `ie`, `ielt9`, `edge`, `webkit`, `android`, `android23`, `androidStock`, `opera`, `gecko`, `phantom`, `opera12`, `win`, `ie3d`, `webkit3d`, `gecko3d`, `any3d`, `mobileWebkit`, `mobileWebkit3d`, `msPointer`, `mobileOpera`, `mobileGecko`, `passiveEvents`, `canvas`, `svg`, `vml`, `inlineSvg`, `touch`, `touchNative`
+- Flags like `L_NO_TOUCH` and `L_DISABLE_3D` are also gone
+
+### DomUtil
+- `setPosition` and `getPosition` no longer rely on `_leaflet_pos`
+- Removed: `getStyle`, `testProp`, `empty`, `remove`, `TRANSITION`, `TRANSITION_END`, `TRANSFORM`, `removeClass`, `setOpacity`, `addClass`, `setClass`, `getClass`, `hasClass`
+- `enableTextSelection` and `disableTextSelection` now use standard `userSelect` style (no vendor prefixes anymore)
+- `setTransform` drops vendor prefixed style `transform` too
+
+### Util
+- Removed: `trim`, `create`, `isArray`, `bind`, `requestAnimFrame`, `cancelAnimFrame`, `extend`, `getParamString`
```

Comment:
Explicitly state the replacements with link to MDN for all of those functions? It makes the migration easier for everyone instead. :)